sesman/session.c

```diff
@@ -837,12 +889,25 @@ session_start_fork(tbus data, tui8 type, struct SCP_CONNECTION *c,
                     "auth_stop_session and auth_end from pid %d", g_getpid());
                 auth_stop_session(data);
                 auth_end(data);
+
+                LOG(LOG_LEVEL_INFO, "Cleaning up session: terminating Xserver "
+                    "(pid %d)", display_pid);
```

Comment:
What about also record the display number?

Reply:
done